src/views/lighttable.c

```diff
@@ -1879,12 +1907,21 @@ static gboolean _expose_compute_slots(dt_view_t *self, int32_t width, int32_t he
 
   GList *slots = NULL;
 
+  // reinit size and positions
+  int i = 0;
```

Comment:
to be remove

Reply:
yes, I didn't like that too, but it was here before... time to remove !